cEP-0005.md

```diff
@@ -49,9 +49,9 @@ paragraph:
   not a program and must remain flow insensitive.
 - To specify, what analysis should be run the user will not have to care about
   bears. Instead, an `aspects` setting will be provided allowing the user to
-  specify what aspects of their code should be analyzed. Aspects could be
+  specify what aspects of their code should be analyzed. aspects could be
   `spelling` or `smells` and are namespaced such that `redundancy.clones` will
```

Comment:
aspect names should also be in `CamelCase` like their class names here. To stay consistent throughout the document...